app/src/main/java/com/android/partagix/ui/screens/EditAccount.kt

```diff
@@ -157,20 +181,111 @@ fun EditAccount(
+                            imageVector = Icons.Default.KeyboardArrowDown,
+                            contentDescription = null)
+                      }
+                      Text("Contact information ")
+                    }
+
+                if (contact) {
+                  Spacer(modifier = Modifier.height(8.dp))
+                  Text(text = "Favorite contact methods", modifier = Modifier.padding(16.dp, 0.dp))
+                  Spacer(modifier = Modifier.height(4.dp))
+                  Row(
+                      modifier = Modifier.fillMaxWidth().padding(start = 16.dp, end = 32.dp),
+                      horizontalArrangement = Arrangement.Absolute.SpaceEvenly) {
+                        Row(
+                            horizontalArrangement = Arrangement.Absolute.Left,
+                            verticalAlignment = Alignment.CenterVertically) {
+                              Checkbox(
+                                  checked = favorite[0],
+                                  onCheckedChange = { favorite[0] = !favorite[0] },
+                                  enabled = email.contains("@"),
+                                  modifier = Modifier.testTag("emailCheckbox"))
+                              Text("Email", fontSize = 12.sp)
+                            }
+                        Spacer(modifier = Modifier.width(2.dp))
+                        Row(
+                            horizontalArrangement = Arrangement.Absolute.Left,
+                            verticalAlignment = Alignment.CenterVertically) {
+                              Checkbox(
+                                  checked = favorite[1],
+                                  onCheckedChange = { favorite[1] = !favorite[1] },
+                                  enabled = phoneNumber.isNotEmpty(),
+                                  modifier = Modifier.testTag("phoneNumberCheckbox"))
+                              Text("Phone number", fontSize = 12.sp)
+                            }
+                        Spacer(modifier = Modifier.width(2.dp))
+                        Row(
+                            horizontalArrangement = Arrangement.Absolute.Left,
+                            verticalAlignment = Alignment.CenterVertically) {
+                              Checkbox(
+                                  checked = favorite[2],
+                                  onCheckedChange = { favorite[2] = !favorite[2] },
+                                  enabled = telegram.isNotEmpty(),
+                                  modifier = Modifier.testTag("telegramCheckbox"))
+                              Text("Telegram", fontSize = 12.sp)
+                            }
+                      }
+
+                  Spacer(modifier = Modifier.height(10.dp))
+
+                  TextField(
+                      modifier = Modifier.fillMaxWidth().padding(16.dp, 0.dp).testTag("email"),
+                      value = email,
+                      onValueChange = { email = it },
```

Comment:
Here (and for the 2 other fields too), it would be great that if the user erases the entire input field, than the checkbox is unchecked and removed from `favorite[]`
```suggestion
                      onValueChange = { 
                         email = it 
                         if (it.isEmpty()) { favorite[0] = !favorite[0] }
                      },
```